fastlane/Fastfile

```diff
@@ -22,6 +22,26 @@ platform :ios do
     sh 'git config --global --unset url."https://github.com/".insteadOf'
   end
 
+  desc "Commit version bump in _versions"
+    lane :commit_sdk_ver_bump do |options|
```

Comment:
It wasn't a part of the scope but let's leave this lane. I'm afraid that we won't be able to use it in the release pipeline because of release bot account reduced permissions for public repos.

Reply:
Updated here [db005c0](https://github.com/rakutentech/ios-inappmessaging/pull/288/commits/db005c03d6ec94e1c30a6a69ced0b17d58232219).